src/main/java/mate/academy/spring/MainApp.java

```diff
@@ -14,21 +18,28 @@ public static void main(String[] args) throws SQLException {
         AnnotationConfigApplicationContext context =
                 new AnnotationConfigApplicationContext(AppConfig.class);
 
+        AuthorService authorService = context.getBean(AuthorService.class);
+        Author tarasShevchenko = new Author("Taras", "Shevchenko");
+        authorService.add(tarasShevchenko);
+        RentService rentService = context.getBean(RentService.class);
+        rentService.rentBook(new User("Sunil", "Bora", "suni.bora@example.com"),
+                new Book("Inland: A Novel", 2019, 25.6));
         BookService bookService = context.getBean(BookService.class);
-        bookService.add(new Book("Some book1", 2018, 2.6));
-        bookService.add(new Book("Some book2", 2017, 12.16));
-        bookService.add(new Book("Some book3", 2019, 22.26));
-        bookService.add(new Book("Some book4", 2020, 32.56));
-
+        Book daisyJonesBook = new Book("Daisy Jones & The Six", 2019, 41.6);
+        bookService.add(daisyJonesBook);
+        bookService.add(new Book("Daisy Jones & The Six", 2019, 35.16));
+        bookService.add(new Book("The Water Cure: A Novel", 2019, 52.26));
+        bookService.add(new Book("My Lovely Wife", 2019, 32.56));
+        User userTemp = new User("Sunil", "Bora", "suni.bora@example.com");
         UserService userService = context.getBean(UserService.class);
-        userService.add(new User("Sunil", "Bora", "suni.bora@example.com"));
+        userService.add(userTemp);
```

Comment:
@TarasHryts userTemp?)
```suggestion
        userService.add(userSunilBora);
```

Reply:
done